apps/mudita-center-e2e/src/specs/overview/kompakt-switching-devices.ts

```diff
@@ -0,0 +1,141 @@
+import { E2EMockClient } from "../../../../../libs/e2e-mock/client/src"
+import {
+  overviewDataWithOneSimCard,
+  overviewDataWithOneSimCard2nd,
+} from "../../../../../libs/e2e-mock/responses/src"
+import OverviewPage from "../../page-objects/overview.page"
+import OverviewKompaktPage from "../../page-objects/overview-kompakt.page"
+import HomePage from "../../page-objects/home.page"
+import tabsPage from "../../page-objects/tabs.page"
+import { kompaktImageRegex } from "../../consts/regex-const"
+import overviewPage from "../../page-objects/overview.page"
+import drawerPage from "../../page-objects/drawer.page"
+
+describe("E2E mock sample - overview view", () => {
+  const firstSerialNumber = "first-serial-number"
+  const secondSerialNumber = "second-serial-number"
+
+  before(async () => {
+    E2EMockClient.connect()
+    //wait for a connection to be established
+    await browser.waitUntil(() => {
+      return E2EMockClient.checkConnection()
+    })
+  })
+
+  after(() => {
+    E2EMockClient.stopServer()
+    E2EMockClient.disconnect()
+  })
+
+  it("Connect device", async () => {
+    E2EMockClient.mockResponse({
+      path: "path-1",
+      body: overviewDataWithOneSimCard,
```

Comment:
Your path-1 device needs to have the same serial number in `addDevice` as it has in `overviewDataWithOneSimCard`